### PR TITLE
Add information about what we want to learn from our potential clients

### PR DESCRIPTION
**Pull request by ionut-mihalache**
Signed-off-by: Ionut Mihalache <ipopescu46@gmail.com>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -746,6 +746,42 @@ <h3 class="heading1"><br>Revenue streams</h3>
...
+    <img src="./resources/problem.png" alt="Bogdan" style="width:20%">
+    <p>
+      In order to gather usefull information about the potential client we thought that it would help to give questionnaires to young people,
+      fresh graduates or bachelor students mainly (an easier segment due to the fact that we can reach the segment faster), people in the business
+      environment (the segment we think that is going to embrace our solution the most) and older people(the segment from which we don't expect the 
+      most clients but we wanted to know how much our idea represents a solution for them too because it is important when creating the MVP).
+    </p>
+    <p>
+      Reaching the first segment of potential clients is easier because the age difference is not that big and the communication is easier. We are 
+      intersted in finding out how they transfer information between them currently, not focusing on things like chatting but more precisely what the
```
Comment (paulstelian97) [inline index.html]: Typo: "interested".

**Review by paulstelian97 — COMMENTED** [review]
Besides the typo everything looks good. Fix it (force push) and I'll merge it if Bogdan doesn't have any comments.

**Review by paulstelian97 — APPROVED** [review]
Would have appreciated you squashing the patches but I guess on this repo it doesn't matter that much. Approved.

File: index.html [hunk 2]
```diff
@@ -746,6 +746,42 @@ <h3 class="heading1"><br>Revenue streams</h3>
...
+      created quite frequentely and we would need to keep up with the pace in order to keep the interest as fresh as possible with easy access to the 
+      newest application after the accounts would be shared.
+    </p>
+    <p>
+      For the second segment we are interested to see if the problem we are trying to solve is actually a problem for them. Because we know we address 
+      to business people we know that the information transfer is more formal(e-mail addresses, calls, conferences) and each information needed for 
+      this kind of communication usually is represented by another account which needs to be somehow shared, process that may take quite a while 
+      considering personal preferences and the evolution of such methods of communication. To reach this segment we could give questionnaires to 
+      people we know from our internships, people from companies that we worked or work for, and ask our acquiantances to spread the word and pass the 
+      questionnaire we creted.
```
Comment (firutibogdan) [inline index.html]: *created

**Pull request merged**